lib/daru/date_time/index.rb

```diff
@@ -226,7 +226,7 @@ def each(&block)
       to_a.each(&block)
     end
 
-    attr_reader :frequency, :offset, :periods
+    attr_reader :frequency, :offset, :periods, :keys
```

Comment:
Why this change?.. For what I can tell, nothing fails without this additional reader.